samples/excel/tables/create-table.yaml

```diff
@@ -0,0 +1,94 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Create a table
+description: Creates a table with four columns and seven rows.
+script:
+  content: |+
+    $('#create-table').click(createTable);
+
+    /* Create a new table with some sample data */
+    function createTable() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // Queue a command to add a new sheet
+                sheet = context.workbook.worksheets.add();
+
+                // Call the helper to create the table
```

Comment:
If the helper function has a good name then you don't need this comment to basically say the same thing.